### DISCUSSION TITLE
Calendar powered streaming site

### DISCUSSION THREAD

**Discussion by JayGreentree-Codes** [solspace/craft-calendar#231] [Show & Tell]
Solspace calendar is powering both the countdown and right side scheduled list
![2023-05-30 08_10_43-JayGreentree Live](https://github.com/solspace/craft-calendar/assets/215971/cff2cfaa-8779-4872-a0d9-174d14af7e17)

**Comment by kjmartens** [comment]
Very neat! Thanks for sharing @JayGreentree 🙂

**Reply by JayGreentree-Codes** [reply]
I need to figure out how to use the calendar ics for the "remind me" button still but I'm getting there slowly.
